src/SplitPackages/SplitPackages.csproj

```diff
@@ -4,9 +4,10 @@
   <PropertyGroup>
     <Description>Copies NuGet packages form a given source folder into a specific set of folders based on a CSV file.</Description>
     <VersionPrefix>1.0.1</VersionPrefix>
-    <TargetFrameworks>netcoreapp1.0;net451</TargetFrameworks>
+    <TargetFrameworks>netcoreapp1.0</TargetFrameworks>
     <DebugType>portable</DebugType>
     <OutputType>exe</OutputType>
+    <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
```

Comment:
Because we need a executable: https://github.com/aspnet/Coherence/blob/dev/build/repo.targets#L50